onchain/rollups/contracts/dapp/CartesiDApp.sol

```diff
@@ -199,43 +190,20 @@ contract CartesiDApp is
         bytes calldata _notice,
         Proof calldata _proof
     ) external view override {
-        bytes32 epochHash;
-        uint256 firstInputIndex;
-        uint256 lastInputIndex;
+        // calculate the input index, and check if it is inside the range
```

Comment:
Even though we are not using the input index, we need to calculate it (internally), right?

Reply:
Yes, that's what it does. We could simply say what it is used for, because the calculation is not relevant here. But I'm ok with the current way too :)